angular.json

```diff
@@ -184,8 +183,7 @@
                     "options": {
                         "tsConfig": "apps/docs-e2e/tsconfig.e2e.json",
                         "exclude": [
-                            "**/node_modules/**",
```

Comment:
Why you removed node_modules matcher from excludes? I don't think we want to run linter over packages.

Reply:
`!apps/docs-e2e/**` as I see, this string means that we exclude everything except `apps/docs-e2e`

Reply:
Makes sense 😋